module/zfs/vdev.c

```diff
@@ -1622,39 +1658,67 @@ vdev_uses_zvols(vdev_t *vd)
 	return (B_FALSE);
 }
 
-void
-vdev_open_children(vdev_t *vd)
+/*
+ * Returns B_TRUE if the passed child should be opened.
+ */
+static boolean_t
+vdev_default_open_children_func(vdev_t *vd)
+{
```

Comment:
If we can settle on a common convention to apply everywhere that's fine with me.  In this case a `default_op` felt like it made sense rather than doing something like passing a NULL.